### DISCUSSION TITLE
Having two sections in CHANGELOG

### DISCUSSION THREAD

**Discussion by rousseldenis** [GrandOrgue/grandorgue#1189] [Developers]
@larspalo @oleg68 

Sometimes, it's difficult for users to distinguish Features than Technical fixes.

A change in our procedures could be to have two sections like:

[Features]
- Feature X

[Fixes]
- Fix Y

This can help for instance to collect all new features we've added between versions and highlight those ones in particular when sharing version updates.

What do you think of ?

**Comment by larspalo** [comment]
I'm not so sure it really would help having two different sections. It's a long file and the chronological approach, with the most recent at top, is likely better, but I do think that we could start prefixing the lines with (Fix) or (Feature) just to make things more obvious.

**Reply by oleg68** [reply]
But now we already prefixing lines with Fixed and Added. Is it not sufficient?

**Reply by larspalo** [reply]
Sure, _some_ of the lines have such prefixes, and I'm not the one raising the issue. However, it's not really systematic and certainly not as clear as having the line start with [Fix] or [Feature], or perhaps utilizing the markdown in some sensible way. But again, I don't think this really is all that crucial.

**Reply by oleg68** [reply]
@rousseldenis could you reformat several last versions and post here, how do you want that they'd look as?

**Reply by rousseldenis** [reply]
Yes, this is not crucial, but makes users easier review of what are the changes their interest. Because, my feelings are that's a recurrent discussion in some forums that features are not added and users passed by new upgrades.

This was raised because a new feature will be added in the next version (Crescendo Override) and I don't want people miss it.

**Reply by rousseldenis** [reply]
# 3.7.0 (2022-08-11)

## Features
- Reorganised the Organs tab of the Settings dialog https://github.com/GrandOrgue/grandorgue/issues/1119
- Added capability of restoring organ settings values for all subobjects https://github.com/GrandOrgue/grandorgue/issues/1119


## Fixes
- Deleting an organ in the Organ tab of the Settings dialog causes also deleting it's cache and all it's presets https://github.com/GrandOrgue/grandorgue/issues/1049
- Fixed packaging for OSx https://github.com/GrandOrgue/grandorgue/issues/1135

**Reply by oleg68** [reply]
But only one section will be present for bugfix releases (with non-xero third number of version). Is it acceptable?

**Comment by oleg68** [comment]
@larspalo Will we do this?

**Comment by larspalo** [comment]
I don't mind.

Den ons 14 sep. 2022 13:41Oleg Samarin ***@***.***> skrev:
